docker/build/cicd.centos.Dockerfile

```diff
@@ -1,12 +1,15 @@
 ARG ARCH="amd64"
 
 FROM ${ARCH}/centos:7
-ENV GOLANG_VERSION 1.12
-ARG ARCH="amd64"
+
 RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
     yum update -y && \
-    yum install -y autoconf wget awscli git gnupg2 nfs-utils python36 sqlite3 boost-devel expect jq libtool gcc-c++ libstdc++-devel libstdc++-static rpmdevtools createrepo rpm-sign bzip2 which ShellCheck
+    yum install -y autoconf wget awscli git gnupg2 nfs-utils python3-devel sqlite3 boost-devel expect jq \
+    libtool gcc-c++ libstdc++-devel libstdc++-static rpmdevtools createrepo rpm-sign bzip2 which ShellCheck \
+    libffi-devel openssl-devel
 WORKDIR /root
+ARG ARCH="amd64"
```

Comment:
```suggestion
ARG ARCH
```

Reply:
Do you not like having a default for our arch? That's ok, but it just means that whenever this Dockerfile is used, we'll have to use the --build-arg option

Reply:
If you pass without arg it'll just pull the value from before the FROM line.